### ISSUE TITLE
Cannot select filter value.

### ISSUE THREAD

**Issue by edhgoose** [PostHog/posthog#16123]
## Bug description

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  

I want to add a filter to the Events Explorer. I select the key I want, but when I click to set the value (e.g. true/false) the box disappears.

I therefore can't input a value.

*If you are on PostHog Cloud it would be really valuable if you can share any links where the problem occurs. This speeds up our ability to troubleshoot tremendously.* 

I'm on https://eu.posthog.com/events.

## How to reproduce

1. Go to Events Explorer
2. Add a person property filter
3. Choose the property you want
4. Try to select a value
5. 💥 

https://github.com/PostHog/posthog/assets/1108173/7f3daf34-ffdb-4336-a832-f04a5801d00e

## Environment

- [x] PostHog Cloud EU, project ID: 3310

**Comment by Twixes** [comment]
Hey @edhgoose! Thanks for the report, although we're having trouble reproducing this. Are you still experiencing this? If not, I'll close this in a few days

**Comment by edhgoose** [comment]
Yes, I'm afraid I can still reproduce @Twixes . I can reproduce in both Firefox and Chrome.

Chrome: Version 114.0.5735.133 (Official Build) (arm64)
Firefox: Firefox 114.0.1 (20230608214645)

**Comment by krzysztofwolowicz** [comment]
@Twixes Unfortunately I have a similar problem. I get an error 500 when filtering, this is the payload sent form frontend:

`{
    "query": {
        "kind": "EventsQuery",
        "select": [
            "*",
            "event",
            "person",
            "coalesce(properties.$current_url, properties.$screen_name) -- Url / Screen",
            "properties.$lib",
            "timestamp"
        ],
        "orderBy": [
            "timestamp DESC"
        ],
        "after": "all",
        "event": "Signed Up",
        "properties": [
            {
                "key": "email",
                "value": "is_set",
                "operator": "is_set",
                "type": "event"
            }
        ]
    }
}`

response:

`{
    "type": "server_error",
    "code": "error",
    "detail": "A server error occurred.",
    "attr": null,
    "error_event_id": "2a6793dd4b4b4134a4fdd3c97085d56d"
}`
